From ff4706f6c42165f05bad53e1082eac1fae3efca1 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Mon, 27 Sep 2010 10:30:22 +0800
Subject: [PATCH] ENGR00131616-4 defconfig: enable otg mode and real-time PM

1. Enable otg mode at defconfig for otg port
2. The 2.6.35 usb core uses real-time power management to deal with
auto-suspend

Signed-off-by: Peter Chen <peter.chen@freescale.com>
---
 arch/arm/configs/imx5_defconfig |   15 ++++++++++-----
 1 files changed, 10 insertions(+), 5 deletions(-)

diff --git a/arch/arm/configs/imx5_defconfig b/arch/arm/configs/imx5_defconfig
index 74ccd27..1dbaad0 100644
--- a/arch/arm/configs/imx5_defconfig
+++ b/arch/arm/configs/imx5_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.35.3
-# Sat Sep 25 14:12:18 2010
+# Sun Sep 26 15:36:27 2010
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -416,7 +416,7 @@ CONFIG_SUSPEND_NVS=y
 CONFIG_SUSPEND=y
 CONFIG_SUSPEND_FREEZER=y
 CONFIG_APM_EMULATION=y
-# CONFIG_PM_RUNTIME is not set
+CONFIG_PM_RUNTIME=y
 CONFIG_PM_OPS=y
 CONFIG_ARCH_SUSPEND_POSSIBLE=y
 CONFIG_NET=y
@@ -1695,7 +1695,8 @@ CONFIG_USB=y
 # CONFIG_USB_DEVICEFS is not set
 CONFIG_USB_DEVICE_CLASS=y
 # CONFIG_USB_DYNAMIC_MINORS is not set
-# CONFIG_USB_OTG is not set
+CONFIG_USB_SUSPEND=y
+CONFIG_USB_OTG=y
 # CONFIG_USB_OTG_WHITELIST is not set
 # CONFIG_USB_OTG_BLACKLIST_HUB is not set
 # CONFIG_USB_MON is not set
@@ -1708,8 +1709,12 @@ CONFIG_USB_DEVICE_CLASS=y
 # CONFIG_USB_C67X00_HCD is not set
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_EHCI_ARC=y
-# CONFIG_USB_EHCI_ARC_OTG is not set
+CONFIG_USB_EHCI_ARC_OTG=y
 # CONFIG_USB_STATIC_IRAM is not set
+# CONFIG_USB_EHCI_FSL_MC13783 is not set
+# CONFIG_USB_EHCI_FSL_1301 is not set
+# CONFIG_USB_EHCI_FSL_1504 is not set
+CONFIG_USB_EHCI_FSL_UTMI=y
 CONFIG_USB_EHCI_ROOT_HUB_TT=y
 # CONFIG_USB_EHCI_TT_NEWSCHED is not set
 # CONFIG_USB_EHCI_MXC is not set
@@ -1840,7 +1845,7 @@ CONFIG_USB_OTG_UTILS=y
 # CONFIG_USB_GPIO_VBUS is not set
 # CONFIG_USB_ULPI is not set
 # CONFIG_NOP_USB_XCEIV is not set
-# CONFIG_MXC_OTG is not set
+CONFIG_MXC_OTG=y
 CONFIG_MMC=y
 # CONFIG_MMC_DEBUG is not set
 CONFIG_MMC_UNSAFE_RESUME=y
-- 
1.5.4.4

